### ISSUE TITLE
QUIC: Performance issue.

### ISSUE THREAD

**Issue by scw00** [apache/trafficserver#5548]
There is a  performance gap between ats and ngtcp2. 

ATS is drove by peer's packets. This means even though we have the data to sent ats still pending for waiting for the next packets.

**Comment by bryancall** [comment]
This was about the original in-house QUIC implementation's send/event loop (ATS waiting on the peer's packets before sending). That entire stack was removed in #10703 (2023) and replaced with a quiche-based implementation that drives sending on its own timer, so the specific scheduling gap described here no longer exists in the code. Closing as stale; if you measure a QUIC performance gap on the current stack, please open a new issue with the numbers.

_Update: the original QUIC stack was removed as of the 10.0.0 release, so this is moot from 10.0.0 onward (current: 10.1.2)._
